app/views/candidate_interface/adviser_sign_ups/interruptions/show.html.erb

```diff
@@ -0,0 +1,26 @@
+<% content_for :title, title_with_error_prefix(t('page_titles.new_adviser_sign_up'), @adviser_interruption_form.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_details_path, t('.back')) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-three-quarters">
+      <%= form_with(model: @adviser_interruption_form, url: candidate_interface_update_adviser_sign_ups_interruption_response_path(@application_form.id), method: :patch) do |f| %>
```

Comment:
We could avoid setting `@application_form` in the controller and just doe `@adviser_interruption_form.application_form.id` since this is the only place it's used. This is not a blocker, just might make the controller a bit less verbose.